public/app/features/commandPalette/actions/staticActions.ts

```diff
@@ -100,6 +101,33 @@ function getGlobalActions(): CommandPaletteAction[] {
       priority: PREFERENCES_PRIORITY,
     },
   ];
+
+  if (process.env.NODE_ENV === 'development') {
+    // eslint-disable @grafana/no-untranslated-strings
+    const currentState = currentMockApiState();
+    const mockApiAction = currentState ? 'Disable' : 'Enable';
+    actions.push(
+      {
+        id: 'preferences/dev',
```

Comment:
Do you need this as a parent when you only have one child, which already has its own section?

Reply:
Good point, I can just add a `section` property and not have this separate one, thanks :+1: